orgs/opentelekomcloud-docs/data.yaml

```diff
@@ -0,0 +1,24 @@
+---
+branch_protection_templates:
+  zuul:
+    required_status_checks:
+      strict: false
+      checks:
+        - context: "eco/check"
+          app_id: 11628
+    enforce_admins: true
+    required_pull_request_reviews:
+      dismissal_restrictions:
+        users: []
+        teams: []
+      dismiss_stale_reviews: true
+      require_code_owner_reviews: false
+      required_approving_review_count: 1
+    restrictions:
+      users: []
+      teams: []
+      apps:
+        - otc-zuul
+    required_linear_history: false
+    allow_force_pushes: false
```

Comment:
maybe `true` in all `data.yaml`?

Reply:
it can be true, but we want to explicitly forbid force pushes to the protected branches